### PR TITLE
Add support for specifying the semver-range

### LINKED ISSUE #2: Add a way to pass in tag/version
Need to be able to do the equivalent 

``await latestVersion('npm', {version: 'latest-5'})``

### PR DESCRIPTION
**Pull request by LitoMore**
It's a reopened PR for #3 

### Changes

- Add support for specifying the semver-range by passing the flag `--range`.
- ~Upgrade devDependencies to the latest version.~
- Requrie Node.js 10.
- Add .npmrc file to disable npm lockfile.
Fixes #2

Resolve #2

### CODE REVIEW

**Comment by LitoMore** [comment]
@sindresorhus Can I update other dependencies to the latest version in this PR? And upgrade runtime requirement to Node.js 10?

File: cli.js [hunk 1]
```diff
@@ -7,14 +7,37 @@ const cli = meow(`
 	Usage
 	  $ latest-version <package-name>
 
+	Options
+	  -v         Ouput the version of this package
+	  --version  Specify which semver-range to use to find the latest version
```
Comment (sindresorhus) [inline cli.js]: This is outdated, and the `-v` flag is no longer needed now.

File: readme.md [hunk 2]
```diff
@@ -18,9 +18,16 @@ $ latest-version --help
   Usage
     $ latest-version <package-name>
 
+  Options
+    -v         Ouput the version of this package
+    --range    Specify which semver-range to use to find the latest version
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
    --range    Specify which semver range to use to find the latest version
```

We should make it clear that dist-tags are supported too.

**Comment by sindresorhus** [comment]
> Can I update other dependencies to the latest version in this PR? And upgrade runtime requirement to Node.js 10?

Sure

**Comment by LitoMore** [comment]
@sindresorhus Updated.

**Comment by LitoMore** [comment]
From the example:

```
$ latest-version ava --range=next
```

The dist-tag `next` has been removed from [ava](https://www.npmjs.com/package/ava). Do you have any other recommended packages for this example?

**Comment by sindresorhus** [comment]
You missed this part:

> and the -v flag is no longer needed now.

**Comment by sindresorhus** [comment]
> Do you have any other recommended packages for this example?

No, I don't usually use dist-tags.

File: readme.md [hunk 3]
```diff
@@ -18,9 +18,16 @@ $ latest-version --help
   Usage
     $ latest-version <package-name>
 
+  Options
+    -v         Ouput the version of this package
```
Comment (LitoMore) [inline readme.md]:
```suggestion
```

File: readme.md [hunk 4]
```diff
@@ -18,9 +18,15 @@ $ latest-version --help
   Usage
     $ latest-version <package-name>
 
+  Options
+    --range    Specify which semver range to use to find the latest version
```
Comment (sindresorhus) [inline readme.md]: This is inconsistent with the actual output. Please just copy-paste the output to get it correctly.

Comment (LitoMore) [reply readme.md]: Good point, thanks!

Updated.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by LitoMore** [comment]
Conflict resolved. By the way, I think we can drop support for Node.js 8.

**Pull request merged**